paasta_tools/setup_istio_mesh.py

```diff
@@ -103,7 +103,9 @@ def get_existing_kubernetes_service_names(kube_client: KubeClient) -> Set[str]:
     return {
         item.metadata.name
         for item in service_objects.items
-        if item.annotations.get(paasta_prefixed("managed_by")) == "setup_istio_mesh"
+        if item.metadata.annotations is not None
+        if item.metadata.annotations.get(paasta_prefixed("managed_by"))
```

Comment:
`and if`?

Reply:
it's a nested condition not to avoid raising `NoneType` attribute error